ui/raidboss/data/07-dt/raid/r6s.ts

```diff
@@ -404,6 +436,89 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'R6S Cactus Spam Pattern Identifier',
+      type: 'StartsUsingExtra',
+      netRegex: { id: 'A6A1' },
+      condition: (_data, matches) => {
+        const matchX = parseFloat(matches.x);
+        const matchY = parseFloat(matches.y);
+        const cactus = findCactus(cactusSpamPatterns, matchX, matchY);
+        return cactus !== undefined;
+      },
+      suppressSeconds: 9999,
+      infoText: (_data, matches, output) => {
+        const matchX = parseFloat(matches.x);
+        const matchY = parseFloat(matches.y);
+        const cactus = findCactus(cactusSpamPatterns, matchX, matchY);
+        if (cactus === undefined)
+          return;
+
+        switch (cactus.id) {
+          case '1':
+            return output.pattern1!();
+          case '2':
+            return output.pattern2!();
+          case '3':
+            return output.pattern3!();
+          case '4':
+            return output.pattern4!();
+        }
+
+        return output.unknown!();
+      },
+      outputStrings: {
+        unknown: Outputs.unknown,
+        pattern1: {
+          en: 'Cactus Pattern 1',
+        },
+        pattern2: {
+          en: 'Cactus Pattern 2 (bad)',
+        },
+        pattern3: {
+          en: 'Cactus Pattern 3',
+        },
+        pattern4: {
+          en: 'Cactus Pattern 4',
+        },
+      },
+    },
+    {
+      id: 'R6S Cactus Quicksand Pattern Identifier',
+      type: 'StartsUsingExtra',
+      netRegex: { id: '9A2C' },
+      condition: (_data, matches) => {
+        const matchX = parseFloat(matches.x);
+        const matchY = parseFloat(matches.y);
+        const cactus = findCactus(cactusQuicksandPatterns, matchX, matchY);
+        return cactus !== undefined;
+      },
+      suppressSeconds: 9999,
+      infoText: (_data, matches, output) => {
+        const matchX = parseFloat(matches.x);
+        const matchY = parseFloat(matches.y);
+        const cactus = findCactus(cactusQuicksandPatterns, matchX, matchY);
+        if (cactus === undefined)
+          return;
+
+        switch (cactus.id) {
+          case 'dirNE':
+          case 'dirSE':
+          case 'dirSW':
+          case 'dirNW':
+            return output.cactus!({ dir: output[cactus.id]!() });
+        }
+
+        return output.unknown!();
+      },
+      outputStrings: {
+        unknown: Outputs.unknown,
+        ...Directions.outputStrings8Dir,
+        cactus: {
+          en: 'Cactus ${dir}',
```

Comment:
```suggestion
          en: 'Danger Cactus ${dir}',
```
I agree with xiashtra that calling this is the best way to handle it for now. We can probably bolt on a configuration option later on to allow for the Latte "healers 1 CW/CCW" setup, or the "party always goes to the first safespot CCW from north, healers fill from there" strategy.